### DISCUSSION TITLE
How to correctly proactively call non-static methods?

### DISCUSSION THREAD

**Discussion by lakahuha** [vfsfitvnm/frida-il2cpp-bridge#527] [Q&A]
Why can I get the correct result when calling a non-static method in the implementation, but I can't get the correct result when calling it outside the implementation with the same parameters?

**Comment by thinhbuzz** [comment]
You need to find a way to get an instance of that class.

**Reply by lakahuha** [reply]
I have got an instance of that class, when I calling the method it works, but the result is not the same to when I calling  in the implementation.
